marimba-web/Models/Club.cs

```diff
@@ -1,12 +1,176 @@
-﻿using System;
+﻿using CsvHelper;
+
 using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
 using System.Linq;
-using System.Threading.Tasks;
+using System;
+using System.Net.Mail;
 
 namespace marimba_web.Models
 {
-    public static class Club
+    public class Club
     {
+        private List<Term> termList;
+
+        /*
+         * Write list of eligible electors to CSV.
+         */ 
+        public void WriteEligibleElectorsToCsv(string pathToCsv)
+        {
+            List<Elector> electorList = GetEligibleElectors();
+            WriteElectorsToCsv(electorList, pathToCsv);
```

Comment:
Does this need to be its own method?  I don't think anyone else would be using it.